src/spas/libs/member-management/domain/src/lib/user-service-interface.ts

```diff
@@ -0,0 +1,6 @@
+import { Observable } from 'rxjs';
+import { User } from './models/user-profile';
+
+export interface UserInterface {
```

Comment:
interface can be renamed to MemberManagement, will contain methods (and down the road maybe props) associated with managing users, like getting, updating, adding, and deleting. file can also be renamed to member-management.interface.ts